lib/run.js

```diff
@@ -82,3 +82,23 @@ function isProfileName (profile) {
   }
   return !/[\\\/]/.test(profile);
 }
+
+/**
+ * Concat binary arguments to arguments array.
+ *
+ * @param {Array} args - arguments
+ * @param {string|Array} binArgs - binary arguments
+ * @return {Array} - arguments
+ */
+function concatBinArgs (args, binArgs) {
+  var removeQuotes = function (arg) {
+    var quotedArg = /^\s*(?:"(.*)"|'(.*)')\s*$/.exec(arg);
+    return quotedArg ? (quotedArg[1] || quotedArg[2]) : arg;
+  };
+  args = Array.isArray(args) ? args : [];
+  if (typeof binArgs === "string" || binArgs instanceof String) {
+    binArgs = binArgs.match(/(?:^|\s)(?:\"(?:[^"]|\\\")*\"|\'(?:[^']|\\\')*\')(?=\s|$)|(?:[^"'\s]|\"|\')+/g);
```

Comment:
could you add a comment about what this is doing? Is it to handle escaped and double escaped quotes?